doc/source/methods/ProtoSelect.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<ol><li>Import missing (useful for completeness to show where the function comes from)</li><li>Link to API docs for a full description</li><li>"meta-data" -&gt; "metadata"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>